vision-zero

```diff
@@ -177,7 +177,7 @@ def replicateDb(args):
 
     drop_database_command = [
         "psql",
-        "--command=DROP DATABASE " + os.environ["POSTGRES_DB"] + "",
+        "--command=DROP DATABASE " + os.environ["POSTGRES_DB"] + " with (force)",
```

Comment:
I needed to make a commit, any commit, to let me open up this WIP PR, so here's a tiny QOL change to the `vision-zero` program that I'd like to tag-along with this PR.